### ISSUE TITLE
(electric) Power limiter

### ISSUE THREAD

**Issue by YourLocalPlonker** [studio-minus/ppg-bugs-and-requests#1830]
**Feature description**
A device that regulates the amount of electric power being sent trough a conducting wire (from 0-1000 like on the power gauge) to an object that is not a power source

**Feature purpose**
You could adjust the amount of power you need precisely which currently is not really possible as the power sources have always the same constant amount of electricity which is not adjustable.

**Additional context**
-

**Comment by Drogo-man** [comment]
Like the resistor? But the resistor in the game is really wonky and outdated and should def be updated.

**Comment by YourLocalPlonker** [comment]
> Like the resistor? But the resistor in the game is really wonky and outdated and should def be updated.

Yeah that thing. I completely forgot about that. But yeah it is kind of crappy and should be updated
